### PR TITLE
 Use reviewed & unreverted edits as good for fiwiki

### PR DESCRIPTION
**Pull request by halfak**
Please squash on merge.

### CODE REVIEW

**Comment by adamwight** [comment]
This has a few flake8 errors and merge conflicts.

File: Makefile [hunk 1]
```diff
@@ -1441,8 +1441,7 @@ datasets/fiwiki.flaggedrevs_autolabeled.210k_2017.json: \
 datasets/fiwiki.flaggedrevs_autolabeled_unreverted.210k_2017.json: \
 		datasets/fiwiki.flaggedrevs_autolabeled.210k_2017.json
 	cat $< | \
-	python oneoff-fiwiki.flaggedrevs/filter_data.py \
-		autolabel.review_reason --unequal="reverted edit" > $@
+	grep -v '"review_reason": "reverted edit"' > $@
```
Comment (adamwight) [inline Makefile]: lol

File: Makefile [hunk 2]
```diff
@@ -1453,45 +1452,21 @@ datasets/fiwiki.labeled_revisions.20k_2016.json: \
 		datasets/fiwiki.autolabeled_revisions.20k_2016.json
 	./utility merge_labels $^ | shuf > $@
 
-datasets/fiwiki.flaggedrevs_training.65k.json: \
+datasets/fiwiki.combined_labeled_revisions.w_cache.230k_2016.json.bz2: \
```
Comment (adamwight) [inline Makefile]: It's been a while, but wasn't there a reason we were using join_observations?  Are we sure that labeled_revisions and flaggedrevs_autolabeled_unreverted are disjoint?  Is it okay to have some revisions repeated?

**Review by adamwight — COMMENTED** [review]
Nice simplifications!

File: Makefile (hunk 2 above)
Comment (halfak) [reply Makefile]: Hmm... Probably OK unless it is super likely there's a lot of overlap.  I'll check.

**Comment by adamwight** [comment]
I think the other open question was whether this experiment improves classifier health or not—unless you had decided that we should use it regardless?

**Pull request merged**